File: CursoJava/src/Exer/EstruturaCondicional/Exer01.java
package Exer.EstruturaCondicional;

import java.util.Scanner;

public class Exer01 {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite um numero inteiro: ");
		int number = sc.nextInt();
		
		if (number == 0) {
			System.out.println("Zero é um número neutro.");
		} else if (number > 0 ) {
			System.out.println("Seu numero é positivo.");
		} else {
			System.out.println("Seu numero é negativo.");
		}
		
		sc.close();
	}

}

File: CursoCSharp/Exer/EstruturaCondicional/Exer01/Exer01.cs
namespace Exer01;

public class Exer01
{
    static void Main(string[] args)
    {
        Console.WriteLine("Informe um número inteiro: ");
        int number = int.Parse(Console.ReadLine());

        if (number >= 0)
        {
            Console.WriteLine("POSITIVO");
        }
        else
        {
            Console.WriteLine("NEGATIVO");
        }
    }
}